### PR TITLE
Create angelospl.md

### PR DESCRIPTION
**Pull request by angelospl**

### CODE REVIEW

File: docs/_slides/angelospl.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from angelospl!
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
```
Comment (angelospl) [inline docs/_slides/angelospl.md:8]: @bilgesubilgin-signaloid What do you think?

Comment (bilgesubilgin-signaloid) [reply docs/_slides/angelospl.md:8]: LGTM

File: docs/_slides/angelospl.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from angelospl!
```
Comment (bilgesubilgin-signaloid) [inline docs/_slides/angelospl.md:6]:
```suggestion
Greetings from Angelos Plevris!
```

File: docs/_slides/angelospl.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
```
Comment (bilgesubilgin-signaloid) [inline docs/_slides/angelospl.md:3]:
```suggestion
title: "Welcome to my slide deck!"
```

**Review by bilgesubilgin-signaloid — CHANGES_REQUESTED** [review]

**Review by bilgesubilgin-signaloid — APPROVED** [review]
Much better!

**Pull request merged**